index.html

```diff
@@ -45,7 +46,7 @@ <h2 class="p2">LET’S CONNECT</h2>
         <div class="ellipse_13"></div>
         <div class="rectangle_5"></div>
         <div class="ellipse_4"></div>
-
+        <!--list -->
         <ul class="social_media">
           <li>
             <a><img src="Images/Twitter.svg" alt="Twitter" /></a>
```

Comment:
[Optional:] 

- [ ] Your social media icons are not showing because of the wrong path of the folder name. Your image folder name is not started with a capital letter but your image path is started with a capital letter `I`. Please change the image path for all social media icons. 
Example: `<img src="./images/Twitter.svg" alt="Twitter" />`

- [ ] The social media icons are not aligned properly. Kindly fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![a1](https://user-images.githubusercontent.com/70483835/166108833-e550184d-5ff7-497d-a732-437ac71af7f9.png) | ![a2](https://user-images.githubusercontent.com/70483835/166108850-1d12a5b4-e147-43cf-8295-e16e2cf353d0.png)